core/services/keystore/keys/ethkey/models.go

```diff
@@ -13,7 +13,9 @@ type State struct {
 	Disabled   bool
 	CreatedAt  time.Time
 	UpdatedAt  time.Time
-	lastUsed   time.Time
+	// TODO: Make the nonce show in the list/UI
```

Comment:
I'm considering doing this as a followup since it's not that hard and would give a huge QOL improvement